buddyboss/buddyboss-platform
# Fixed #770 Profile widget is not updated when we use Gravatar setting.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes [#770](https://github.com/buddyboss/buddyboss-theme/issues/770) 

### How to test the changes in this Pull Request:

1. Register a user using the email address that is connected with Gravatar 
2. Open the profile page
3. Check the Complete your profile widget
4. See error

### Proof Screenshots or Video

<img width="786" alt="Screenshot 2020-09-04 at 8 37 38 PM" src="https://user-images.githubusercontent.com/2979084/92254412-7c1b1c00-eeee-11ea-812d-40ed887b7539.png">


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
